.gitignore

```diff
@@ -1,13 +1,19 @@
 *#*#
 /puddletag-docs/_build
-*.pyc
-source/releases
-source/deb_build
-source/dist/
 *#*
 puddletag-docs/translations
 .idea/
 venv
 rss_temp*.txt
 rss_temp*.html
-rss_temp*.doctree
\ No newline at end of file
+rss_temp*.doctree
```

Comment:
this looks ok, we may want to grab the "official" python .gitignore from github, i think there's a place where there's a collection of valuable .gitignore files. food for thought